Dockerfile

```diff
@@ -72,8 +74,9 @@ RUN mkdir -p logs/selenium
 # Install required npm dependencies
 
 COPY package.json .
-COPY npm-shrinkwrap.json .
-RUN npm install
+COPY yarn.lock .
+# skips all dev dependencies if NODE_ENV=production.. so..
+RUN yarn install --production=false
```

Comment:
Is this actually what we want? We don't really make a distinction between dev and prod dependencies in package.json right now (the three things in dependencies should really be in devDependencies). They're all necessary to build.